scripts/behavior.js

```diff
@@ -371,7 +371,89 @@ function executeCommand(command, arguments, cooldown, message, settings, utils,
                 }).then(
                     callback(COMMAND_SUCCESS)
                 );
-                
+
+                break;
+
+            case "createrole":
+                if (arguments == null){
+                    callback(COMMAND_MISUSE);
+                    break;
+                }
+                let roleName = escapeArguments(arguments);
+
+                if(isRoleAlreadyExisting(roleName, message.guild)) {
+                    replyToMessage(message, "Role already registered.")
+                        .then(callback(COMMAND_MISUSE));
+                    break;
+                }
+
+                createRole(roleName, message)
+                    .then(callback(COMMAND_SUCCESS), callback(COMMAND_SUCCESS));
+
+                break;
+
+            case "deleterole":guild
+                if (arguments == null){
+                    callback(COMMAND_MISUSE);
+                    break;
+                }
+                let delRoleName = escapeArguments(arguments);
+
+                if(! isRoleAlreadyExisting(delRoleName, message.guild)) {
+                    replyToMessage(message, "Role not registered. Are you sure the spelling is correct?")
+                        .then(callback(COMMAND_MISUSE));
+                    break;
+                }
+
+                deleteRole(delRoleName, message)
+                    .then(callback(COMMAND_SUCCESS), callback(COMMAND_SUCCESS));
+
+                break;
+
+            case "subscribe":
+                if (arguments == null){
+                    callback(COMMAND_MISUSE);
+                    break;
+                }
+                let subscribeRoleName = escapeArguments(arguments);
+
+                if(! isRoleAlreadyExisting(subscribeRoleName, message.guild)) {

```

Comment:
Rename this function to `roleExists`

...also, maybe we shouldn't use the word "role" for that feature. If I check if role "discord moderator" exists, it will return false (because it's not registered by dostya) despite the role exists. so maybe you should call that feature "subscriptions", or I don't know.

Reply:
hmm, probably yes, I'm used to beginning boolean getters with is, but in this case the Exists signals that enough